tests/test_model.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+import colander
+import pirant.models
+
+
+class NewsTest(unittest.TestCase):
+    """
+    Unit tests for the News class model
+    """
+
+    def setUp(self):
+        self.news = pirant.models.News()
+        self.data = {
```

Comment:
rename all self.data to self.testData or self.test_data
will be more clear that this data is just a dummy for testing purpose.